dataproviders/src/test/java/com/paypal/selion/platform/dataprovider/YamlDataProviderTest.java

```diff
@@ -424,16 +420,6 @@ public void testGetDataByIndex_NullData() throws IOException {
         return fetchedStrings;
     }
 
-    public synchronized List<String> transferDataIntoList(Hashtable<String, Object> allUsers) {
-        List<String> fetchedNames = new ArrayList<String>();
-        Set<String> keys = allUsers.keySet();
-        for (String key : keys) {
-            USER user = (USER) allUsers.get(key);
-            fetchedNames.add(user.getName());
-        }
-        return fetchedNames;
-    }
-
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using varargs for methods or constructors which take an array the last parameter.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1249607615/issues/source?bid=2928425&fileBranchId=2937920#l423)